dump26/‏‏claims_new/__test.py

```
@@ -0,0 +1,54 @@
+        qids_tab = dict(qids_1[:max_items])
+        # ---
+        others += sum([x[1] for x in qids_1[max_items:]])
+        # ---
+        print(f"new others: {others}")
+        # ---
+        qids_tab["others"] = others
+    # ---
+    return qids_tab
+
+
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)